samples/bluetooth/central_hids/src/main.c

```diff
@@ -517,22 +517,34 @@ static void auth_cancel(struct bt_conn *conn)
 
 static void auth_done(struct bt_conn *conn)
```

Comment:
I very much appreciate the cleanup you are doing.
Not your fault I know, but would you mind renaming this function to `pairing_confirm`?

Reply:
sure, its aligned now